modules/ticket_selector/assets/ticket_selector.css

```diff
@@ -427,7 +427,14 @@
     top: -2px;
 }
 
-.ee-hidden-ticket-tr {
+
+.tckt-slctr-tbl-tr,
+.tckt-slctr-tkt-details-tr {
+    display: table-row;
+}
+
+.tckt-slctr-tbl-tr.ee-hidden-ticket-tr,
+.tckt-slctr-tkt-details-tr.ee-hidden-ticket-tr {
```

Comment:
is there any chance of using SASS here in the future?

Reply:
afaik there's likely no easy way to convert all of these legacy assets to use a build system that can convert SASS to CSS, so it just isn't worth it imho

the current plan i'm considering is to convert the frontend presentation layer to utilize GB blocks, which we will be able to use SASS for since those will all be developed on the Barista repo.